Core/DefInjected/SkillDef/Skills.xml

```diff
@@ -39,14 +39,14 @@
   <!-- EN: intellectual -->
   <Intellectual.label>Geistige Arbeit</Intellectual.label>
   <!-- EN: Researching and drug synthesis. -->
-  <Intellectual.description>Forschen und Drogen synthetisieren.</Intellectual.description>
+  <Intellectual.description>Das Forschen und Drogen synthetisierung.</Intellectual.description>
```

Comment:
Schreibfehler (Leerzeichen zwischen Drogen und synthetisierung), aber selbst dann klingt es nicht stimmig. Hier wäre so etwas wie "Das Erforschen von Technologien und Synthetisieren von Drogen." sinnvoller.

Reply:
Wollte ich auch gerade schreiben. 
Die restlichen Änderungen finde ich gut, aber bei diesem Satz fehlt noch das bestimmte Etwas (auch vorher schon). Wenn man z. B. schreiben würde "Das Forschen und die Synthetisierung von Drogen", würde es so klingen, als würde auch die Forschung nur mit Drogen zu tun haben.

"Das Erforschen von Technologien und Synthetisieren von Drogen." wäre eine gute Alternative.
Kannst du das bitte noch ändern? Dann nehmen wir das so auf.

Reply:
Ok mach ich

Reply:
@TeiXeR, passt das so?

Reply:
"Das Forschen und syntethisierung von Drogen."
müsste geändert werden in
"Das Erforschen von Technologien und Synthetisieren von Drogen."

Reply:
Ohhh shi* sorry 🤦

Reply:
Jetzt passt alles. ;)